src/components/Input/Input.jsx

```diff
@@ -36,6 +37,7 @@ export default class Input extends React.Component {
             disabled={disabled}
             hasError={hasError}
             maxLength={maxLength}
+            id={name}
```

Comment:
@MayaUribe, I'd probably pass an id instead of using the name prop, since the id has to be unique!